src/common-model/entities.ts

```diff
@@ -81,6 +82,8 @@ export const aliasEntityNames: Record<ClinicalEntitySchemaNames, EntityAlias> =
   biomarker: 'biomarker',
 };
 
+export const queryEntityNames = <EntityAlias[]>Object.values(aliasEntityNames);
```

Comment:
can we keep the casting 'as' syntax over this prefix syntax for consistency?

Reply:
Sure